### DISCUSSION TITLE
Allow osu!lazer clients to play in osu!stable lobbies

### DISCUSSION THREAD

**Discussion by kiwec** [ppy/osu#25778] [Ideas] [answered]
The idea is to have a compatibility layer that allows you to see and join osu!stable lobbies. Lazer-specific features (such as lazer mods, map playlist etc) would be disabled.

Pros:
- Avoids fragmenting the community, makes it easier to switch
- Enables automated tournaments and bots (since there is no room management API yet)
- Temporarily addresses https://github.com/ppy/osu/discussions/16392

Cons:
- Confusing ("why does X not work in this room")
- Would need to open-source parts of the bancho protocol
- Might not be easy to implement (can't estimate, since it's closed source)

**Comment by peppy** [comment, marked as answer]
This is impossible. No one on the team in their right mind would implement such a thing.
